code/modules/reagents/reagents.dm

```diff
@@ -38,6 +38,8 @@
 	var/color = "#000000"
 	var/color_weight = 1
 	var/sanity_gain = 0
+	var/list/taste_tag = list()
+	var/sanity_gain_ingest = 0.5
```

Comment:
Set this to zero by default and add sanity_gain_ingest = 0.5 to the types or subtypes of reagents that interest us.

Reply:
sure
 can do